orchagent/vxlanorch.h

```diff
@@ -19,12 +19,23 @@ enum class MAP_T
     VNI_TO_BRIDGE
 };
 
+typedef enum
+{
+    TUNNEL_MAP_T_VLAN,
+    TUNNEL_MAP_T_BRIDGE,
+    TUNNEL_MAP_T_VIRTUAL_ROUTER,
+} tunnel_map_type_t;
+
 struct tunnel_ids_t
 {
-    sai_object_id_t tunnel_encap_id;
-    sai_object_id_t tunnel_decap_id;
-    sai_object_id_t tunnel_id;
-    sai_object_id_t tunnel_term_id;
+    sai_object_id_t tunnel_vlan_to_vni_encap_id = SAI_NULL_OBJECT_ID;
```

Comment:
Use ```map``` to store mappers IDs in order to optimize the way how it is stored and also to avoid many ```if``` statements in a code that uses these variables.